src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -247,6 +247,7 @@
 		<DataTemplate x:Name="TilesBrowserTemplate" x:DataType="filesystem:ListedItem">
 			<UserControl>
 				<Grid
+					x:Name="MainGrid"
```

Comment:
This can be removed.

Reply:
I refer to it when the picture needs to disappear to make more space for the text.

Reply:
The thumbnail shouldn't be hidden